webapi/webapi-webgl-khronos-tests/webgl/khronos/conformance/more/unit.js

```diff
@@ -58,7 +58,7 @@ Tests = {
   autorun : true,
   message : null,
   delay : 0,
-
+  autoinit: true,

```

Comment:
I'm afraid you used CR-LF at every end of line.
Please use LF in unix style, try to use notepad++(EOL conversion) or "dos2unix" on ubuntu to format these four lines you added.